subsys/partition_manager/Kconfig.template.build_strategy_domain

```diff
@@ -0,0 +1,22 @@
+#
```

Comment:
Why do we need a new template for this ?
If user specify a hex file, then there is nothing to build, but we could still support flashing of that hex file individually.

With this approach: https://github.com/nrfconnect/sdk-zephyr/pull/324
it would be as simple as creating an additional `runners.yaml` file for that hex file.

Reply:
Currently there is no support for providing a hex file for the network core, hence the separate template.

I agree that we should support flashing the network core when we have a solution similar to what is in that PR.

Reply:
Just to ensure that I understand you correctly: You want to have some check in cmake if we are including a hex file for a separate domain, and if yes, then we will generate a new `runners.yaml` the same way that the various `board.cmake` files does it? I assume that the generated `runners.yaml` will be automatically included by west and ninja `flash` targets?

Reply:
Currently there is only support for one `runners.yaml` file. Is it possible to represent the flash-args for two separate hex files inside the same `runners.yaml`? AFAICS this is not supported, and would require a rewrite of `zephyr/cmake/flash/CMakeLists.txt` which I would deem out of scope for this PR. What do you think?

Reply:
@tejlmand  any ideas how it would work with multiple runners?